doc/user/content/sql/system-catalog.md

```diff
@@ -566,50 +566,50 @@ Field          | Type        | Meaning
 `name`         | [`text`]    | The name of the view.
 `definition`   | [`text`]    | The view definition (a `SELECT` query).
 
-### `mz_worker_materialization_frontiers`
+### `mz_worker_compute_frontiers`
 
-The `mz_worker_materialization_frontiers` source describes each worker's
+The `mz_worker_compute_frontiers` source describes each worker's
 frontier for each [dataflow] in the system. The frontier describes the earliest
 timestamp at which the output of the dataflow may change; data prior to that
 timestamp is sealed.
 
 For frontier information aggregated across all workers, see
-[`mz_materialization_frontiers`](#mz_materialization_frontiers).
+[`mz_compute_frontiers`](#mz_compute_frontiers).
 
 Field       | Type       | Meaning
 ------------|------------|--------
-`object_id` | [`text`]   | The ID of the index or materialized view that created the dataflow. Corresponds to [`mz_materializations.object_id`](#mz_materializations).
+`export_id` | [`text`]   | The ID of the index or materialized view that created the dataflow. Corresponds to [`mz_compute_exports.export_id`](#mz_compute_exports).
 `worker_id` | [`bigint`] | The ID of the worker thread hosting the dataflow.
 `time`      | [`mztimestamp`] | The next timestamp at which the dataflow may change.
```

Comment:
We probably want to make the text here consistent with the one for `mz_compute_frontiers`.